.github/workflows/spotless-check.yml

```
@@ -0,0 +1,28 @@
+name: Spotless Check
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)